dev-tools/compose/docker-compose.blockchain.yml

```diff
@@ -1,4 +1,4 @@
-version: "3.9"
+version: '3.9'
```

Comment:
we should probably have some workspace level config for formatting yml files, i get these too if I save without specifying "save without formatting"